src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -363,30 +363,11 @@ Thumbnails =
 # Default: false
 ThumbnailPadding = 
 
-# Some renderers (e.g. Sony Bravia TVs and Blu-ray Disc players) require
-# thumbnails to be represented as JPEGs in the DLNA directory XML. Set this to
-# true to force UMS to fake this representation.
-# Note: this has no effect on real thumbnail file formats or content-type
-# headers, and will only return true if MediaInfo = true
-# Default: false
-ForceJPGThumbnails = 
-
-# Whether the renderer (e.g. Samsung TVs) requires the use of the "res" element
-# instead of the "albumArtURI" element for thumbnails in DLNA reponses.
-# element.
-# Default: false, i.e. use "albumArtURI"
-ThumbnailAsResource = 
-
 # The resolution in pixels of thumbnail images.
 # Default: 320 and 180.
 ThumbnailWidth = 
 ThumbnailHeight = 
 
-# Whether to use ThumbnailHeight as ThumbnailWidth for audio/image thumbnails.
-# Default: false
-SquareAudioThumbnails = 
-SquareImageThumbnails = 
```

Comment:
I have a PS3 and haven't seen any ill-effects. Again, this is made according to the DLNA standard where the thumbnail size is split into different media format profiles, and multiple are offered. The algorithm for choosing which profiles to offer looks at the source image's size, so that no image (or thumbnail) are ever scaled up, but are all offered in scaled-down versions right down to the "default", mandatory ```JPEG_TN``` format which is "no larger than 160x160". Aspect ratio is never altered though, so it will be scaled so that the "biggest" axis is no more than 160 pixels. The larger profiles are offered first, so that "lazy" renderers will hopefully pick the largest version that they can handle.

For video icons the aspect ratio of the thumbnail is decided by the aspect ratio the "source thumbnail" is available in, aka how it is being generated - so they will still be widescreen if generated by UMS. Audio items will typically have the aspect ratio of the cover art, and image thumbnails will have to same aspect ratio as the image itself.